scripts/import-sdvx.js

```diff
@@ -91,7 +117,51 @@ async function main() {
   );
 }
 
-function buildSong(song) {
+function determineDiffClass(song, chartType) {
```

Comment:
This is clutch! I never would have caught on to this little quirk on my own. Question though, is it meaningful to players to distinguish between each of these as separate difficulties when filtering songs for tournament selection? (Just curious, I have no idea as a non-SDVX player)

Reply:
I've only attended one tournament, and they only cared about difficulties and maybe weighted distributions.

I think this is a good granularity to provide still as the sub-classes within the INFINITE type as they tend to include "tricks" not contained in the other standard difficulty classes; some of those tend to be trickier than others, for example I'm decent at GRV charts on average but not so much HVN or VVD charts.